website/project/views/contributor.py

```diff
@@ -378,6 +378,55 @@ def project_manage_contributors(auth, node, **kwargs):
     return {}
 
 
+@no_auto_transaction
+@must_be_valid_project  # injects project
+@must_have_permission(ADMIN)
+@must_not_be_registration
+def project_remove_contributor(auth, **kwargs):
```

Comment:
I find a [`project_removecontributor`](https://github.com/CenterForOpenScience/osf.io/blob/ec285291ea829cf13ff47a8b3631474d35787baa/website/project/views/contributor.py#L171-L212) function also in this file. How does this compare with that?